WcaOnRails/config/routes.rb

```diff
@@ -91,9 +91,12 @@
   get 'admin/delegates' => 'delegates#stats', as: :delegates_stats
 
   get 'robots' => 'static_pages#robots'
+
   get 'server-status' => 'server_status#index'
-  get 'edit-translation' => 'git_hub#edit_translation', as: :edit_translation
-  patch 'update-translation' => 'git_hub#update_translation', as: :update_translation
+
+  get 'translations/status' => 'translations_status#index'
+  get 'translations/edit' => 'git_hub#edit_translation', as: :edit_translation
+  patch 'translations/update' => 'git_hub#update_translation', as: :update_translation
```

Comment:
I agree, it makes sense for this all to become one controller. I've created a separate issue for that here: https://github.com/thewca/worldcubeassociation.org/issues/1333.